### PR TITLE
chore: default policies 

### PR DESCRIPTION
**Pull request by pheel**
- [ ] I wrote tests for the feature
- [ ] I documented the feature if needed

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: botfront/private/defaults/private.dev.yaml [hunk 1]
```diff
@@ -21,16 +21,7 @@ policies:
     max_history: 5
     batch_size: [32, 64]
   - name: RulePolicy
-    enable_fallback_prediction: false
   - name: AugmentedMemoizationPolicy
```
Comment (znat) [inline botfront/private/defaults/private.dev.yaml:24]: Shouldn't `AugmentedMemoizationPolicy` be removed ? Isn't it covered by `RulePolicy`

File: botfront/private/defaults/private.dev.yaml [hunk 2]
```diff
@@ -21,16 +21,7 @@ policies:
...
-  - name: rasa_addons.core.policies.BotfrontDisambiguationPolicy
-    fallback_trigger: 0.30
-    disambiguation_trigger: '$0 < 2 * $1'
-    disambiguation_template: 'utter_disambiguation'
-    n_suggestions: 2
-    excluded_intents:
-      - ^chitchat\..*
-      - ^basics\..*
```
Comment (znat) [inline botfront/private/defaults/private.dev.yaml:33]: If @GuillaumeTech confirms `BotfrontDisambiguationPolicy`  works with latest Rasa, we should leave it

Comment (GuillaumeTech) [reply botfront/private/defaults/private.dev.yaml:33]: Yes the bf disambiguation work fine with the lastest rasa https://www.notion.so/godialogue/8c37f360c4f148fdb89abc7cb106a4d8?v=466d1829d04a4b3eb2e5e3f2d9dfda70&p=9ac119c2e23847ff82dda5344f915008

Comment (pheel) [reply botfront/private/defaults/private.dev.yaml:33]: You haven't tried with both stories and rules.

Comment (pheel) [reply botfront/private/defaults/private.dev.yaml:33]: I had issues when I tried.

File: botfront/private/defaults/private.dev.yaml (hunk 1 above)
Comment (pheel) [reply botfront/private/defaults/private.dev.yaml:24]: `AugmentedMemoizationPolicy` is not deprecated. It handles stories, like `TEDPolicy`.

File: botfront/private/defaults/private.dev.yaml (hunk 2 above)
Comment (GuillaumeTech) [reply botfront/private/defaults/private.dev.yaml:33]: ohh sorry my bad I thought the problem was showing when there was only stories, I'll try with both

Comment (GuillaumeTech) [reply botfront/private/defaults/private.dev.yaml:33]: @pheel @znat No issue when mixing rules and stories !
https://user-images.githubusercontent.com/27060431/107054954-bc810d80-679e-11eb-8f19-e6931e07717a.mov

**Pull request merged**